README.md

```diff
@@ -154,12 +158,12 @@ To specify the number of groups the students should be placed in, use the flag
 `--num-groups`.
 
 ```shell
-pipenv run python3 gatorgrouper/utils/gatorgrouper.py --num-groups 4
+pipenv run python3 gatorgrouper_cli.py --file filepath --num-groups 4
 ```
 
 This indicates that the students should be divided into 4 groups. The number of
 groups should be at minimum 1 and at maximum the number of students to be placed
-into groups. If the number of groups is not specified or is specified as '0' the
+into groups. If the number of groups is not specified or is specified as '3' the
```

Comment:
There is no need to include '3' as an exception here, since it will still be 3...